pdns/recursordist/Makefile.am

```diff
@@ -556,6 +559,9 @@ pdns-recursor@.service: pdns-recursor.service
 	  -e 's!Recursor!& %i!' \
 	  -e 's!RuntimeDirectory=.*!&-%i!' \
```

Comment:
Did we consider removing the `RuntimeDirectory=` directive from the unit when `!HAVE_SYSTEMD_WITH_RUNTIME_DIR_ENV`? I'm guess it currently logs a warning about an unsupported directive?

Reply:
It is my understanding that RuntimeDirectory works, just `RUNTIME_DIRECTORY` is not set in the process environment.

Reply:
Oh, I read that wrong, thanks for the explanation!